From 1a99e66b101cc00954bbc8fd7c248556b9b1601c Mon Sep 17 00:00:00 2001
From: Numan Siddique <nusiddiq@redhat.com>
Date: Mon, 28 Nov 2016 19:38:54 +0530
Subject: [PATCH 4/4] Revert: Configure Keystone Fernet Keys

This requires the commit which changed id
Ic070d160b519b8637997dbde165dbf15275e0dfe present in tripleo-common.
The unercloud may not have these changes. So reverting it.

Signed-off-by: Numan Siddique <nusiddiq@redhat.com>
---
 puppet/services/keystone.yaml | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/puppet/services/keystone.yaml b/puppet/services/keystone.yaml
index fe023a6..d819e04 100644
--- a/puppet/services/keystone.yaml
+++ b/puppet/services/keystone.yaml
@@ -99,12 +99,6 @@ parameters:
   KeystoneCredential1:
     type: string
     description: The second Keystone credential key. Must be a valid key.
-  KeystoneFernetKey0:
-    type: string
-    description: The first Keystone fernet key. Must be a valid key.
-  KeystoneFernetKey1:
-    type: string
-    description: The second Keystone fernet key. Must be a valid key.
   KeystoneLoggingSource:
     type: json
     default:
@@ -162,11 +156,6 @@ outputs:
                 content: {get_param: KeystoneCredential0}
               '/etc/keystone/credential-keys/1':
                 content: {get_param: KeystoneCredential1}
-            keystone::fernet_keys:
-              '/etc/keystone/fernet-keys/0':
-                content: {get_param: KeystoneFernetKey0}
-              '/etc/keystone/fernet-keys/1':
-                content: {get_param: KeystoneFernetKey1}
             keystone::debug: {get_param: Debug}
             keystone::rabbit_userid: {get_param: RabbitUserName}
             keystone::rabbit_password: {get_param: RabbitPassword}
-- 
2.9.3

